src/yang/ietf-qos-action.yang

```diff
@@ -541,22 +544,17 @@ module ietf-qos-action {
 
       leaf burst-value {
         type uint64;
+        units "bytes-per-second";
```

Comment:
burst unit in "bytes" and milli/microsecond.. Why not keep a unit just like rate above

Reply:
Because that is what the RFC suggests

Reply:
you mean the Yang RFC? separate unit gives flexibility to define bust size in bytes and time.

Reply:
No. RFC 2697.

Besides you have a large type defined to be able to define different burst sizes.

Reply:
max-rate is for queues.. check for diffServMaxRateThreshold in rfc 3289.. not sure what you mean by "large type"